scripts/e2e-tests.sh

```diff
@@ -279,19 +271,32 @@ function setup_test_cluster() {
   set +o pipefail
 }
 
-function success() {
+function set_test_return_code() {
```

Comment:
should the argument be documented?

Reply:
Usually I don't document trivial internal functions, but here it goes...